pom.xml

```diff
@@ -430,7 +430,7 @@
         <version.org.jboss.narayana>5.12.0.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.5.Final</version.org.jboss.openjdk-orb>
         <version.org.picketbox>5.0.3.Final-redhat-00007</version.org.picketbox>
-        <version.org.jboss.resteasy>3.15.1.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>4.7.0.Final</version.org.jboss.resteasy>
```

Comment:
should we use the latest 4.7.x here?

Reply:
We will. There are likely some more bug fixes coming in for a 4.7.2 release which we'll upgrade before a WildFly release. The only changes in 4.7.1 are test and POM changes so it won't matter much for WildFly.